src/Card/CardSection.jsx

```diff
@@ -11,15 +11,14 @@ const CardSection = React.forwardRef(({
   children,
   title,
   actions,
-  muted,
```

Comment:
I think we may want to leave the `muted` prop on `Card.Section` to support use cases where consumers want to implement a light variant card with one muted section inside of it. Removing this prop would also technically be a breaking change.

Reply:
True, we returned it, thanks!